addons/hr_timesheet/security/hr_timesheet_security.xml

```diff
@@ -68,9 +68,10 @@
             <field name="model_id" ref="analytic.model_account_analytic_line" />
             <field name="domain_force">[
                 ('project_id', '!=', False),
-                '|',
+                '|', '|',
```

Comment:
we also have to create an upgrade script in hr_timesheet module.

https://github.com/odoo/odoo/blob/master/addons/hr_timesheet_attendance/upgrades/1.1/pre-migrate.py#L1-L14